src/components/BigText/index.module.css

```diff
@@ -17,6 +17,6 @@
 }
 
 .h3 {
```

Comment:
Didn't notice this, my bad, but this should be a `p` on the `BigText`. Looking at Figma, that is text and not headings